src/main/java/com/fundation/logic/view/LoadSavePanel.java

```diff
@@ -0,0 +1,63 @@
+package com.fundation.logic.view;
+
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableModel;
+import java.awt.GridBagLayout;
+import javax.swing.JButton;
+import java.awt.GridBagConstraints;
+import java.awt.Insets;
+
+public class LoadSavePanel extends JPanel {
+    private JTable dataTable;
+
+    /**
+     * Create the panel.
+     */
+    public LoadSavePanel() {
+        GridBagLayout gridBagLayout = new GridBagLayout();
+        gridBagLayout.columnWidths = new int[]{121, 70, 0, 70, 0, 70, 91, 0};
```

Comment:
Magical numbers.

Reply:
Numbers of coordinates in GridBagLayout, please refactor in the next Sprint